include/callbacks.hh

```diff
@@ -12,7 +12,7 @@ int main () {
     alsa_callback::acb test ("hw:2,0,0");
 
     auto cb =
-        [](std::vector<uint16_t> left, std::vector<uint16_t> right) {
+        [](std::vector<int16_t> left, std::vector<int16_t> right) {
```

Comment:
Do these not need to be const references throughout?

Reply:
Good idea. No point copying them when they don't need to be.